src/js/angular/autocomplete/controllers.js

```diff
@@ -257,6 +263,20 @@ function AutocompleteCtrl($scope, $interval, toastr, $repositories, $licenseServ
         pullStatus();
     });
 
+    const initNamespaces = function () {
+        RDF4JRepositoriesRestService.getNamespaces($repositories.getActiveRepository())
+            .success(function (data) {
+                const nss = _.map(data.results.bindings, function (o) {
```

Comment:
I'd use the namespace mapper utility  for this src/js/angular/rest/mappers/namespaces-mapper.js utility

Reply:
Will do also in the RDFRank app.js and push.